core/domain/action_registry_test.py

```diff
@@ -25,19 +25,19 @@
 class ActionRegistryUnitTests(test_utils.GenericTestBase):
     """Test for the action registry."""
 
-    def test_action_registry(self):
+    def test_action_registry(self) -> None:
         """Do some sanity checks on the action registry."""
         self.assertEqual(
             len(action_registry.Registry.get_all_actions()), 3)
 
-    def test_cannot_get_action_by_invalid_type(self):
+    def test_cannot_get_action_by_invalid_type(self) -> None:
         # Testing with invalid action type.
         # Invalid action type raises 'KeyError' with invalid_key
         # as the error message.
-        with self.assertRaisesRegex(KeyError, 'fakeAction'):
+        with self.assertRaisesRegex(KeyError, 'fakeAction'):# type: ignore[no-untyped-call]
```

Comment:
Generally we add 2 spaces to the left of each comment to the right of existing code.

Reply:
Done!